src/Service/Sns/composer.json

```diff
@@ -13,6 +13,7 @@
     "require": {
         "php": "^7.2.5 || ^8.0",
         "ext-SimpleXML": "*",
+        "ext-filter": "*",
```

Comment:
good catch!

Reply:
It was actually added by the code generator. ;)